helix-term/src/ui/mod.rs

```diff
@@ -180,9 +180,7 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
             path.strip_prefix(&root).unwrap_or(path).to_string_lossy()
         },
         move |cx, path: &PathBuf, action| {
-            cx.editor
-                .open(path.into(), action)
-                .expect("editor.open failed");
+            cx.editor.open(path, action).expect("editor.open failed");
```

Comment:
I changed the other occurences in LSP code: instead of an expect it should now use `cx.editor.set_error` to avoid completely crashing